app/assets/stylesheets/styles.css.scss

```diff
@@ -0,0 +1,45 @@
+
+body {
+  background: transparent url("texture.png");
+}
+
+#nav_bar {
+  color: black;
+  background-color: white;
+}
+
+#nav_bar a {
+  color: black;
+}
+
+.home_picture {
+  height: 750px;
+  background: transparent url("homepage.png") no-repeat center -50px;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.